basic python scripts/swapTwoNum.py

```diff
@@ -0,0 +1,14 @@
+# Python program to swap two variables
+
+
+
+a = input('Enter value of a: ')
+b = input('Enter value of b: ')
+
+
+temp = a
+a = b
+b = temp
```

Comment:
Hi. 
You can swap two integers without temp by using xor operator: 
a = a^b
b = b^a
a = a^b

or much better. 
You can swap with tuples:
a, b = b, a